From 89f8d23995b62e79fb1570ae3486301bc17f0866 Mon Sep 17 00:00:00 2001
From: Ao Xu <ao.xu@amlogic.com>
Date: Fri, 7 Aug 2020 20:12:13 +0800
Subject: [PATCH 2/5] drm: disable amcvbs connector [1/1]

PD#SWPL-30680

Problem:
when westeros-soc-drm set invalid mode, there will no signal

Solution:
disable amcvbs connector to avoid 576cvbs

Verify:
u212

Change-Id: Ie9ddaa5b24b32a177f2c590289326c2c6b48e4c0
Signed-off-by: Ao Xu <ao.xu@amlogic.com>
---
 .../dts/amlogic/g12a_s905d2_u200_drm_buildroot.dts  |  2 +-
 .../dts/amlogic/g12a_s905x2_u212_drm_buildroot.dts  |  2 +-
 .../dts/amlogic/g12b_a311d_w400_drm_buildroot.dts   |  2 +-
 .../dts/amlogic/g12b_a311d_w400_drm_buildroot_a.dts |  2 +-
 .../amlogic/g12b_revb_a311d_w400_drm_buildroot.dts  |  2 +-
 arch/arm/boot/dts/amlogic/mesong12a_drm.dtsi        | 21 ---------------------
 arch/arm/boot/dts/amlogic/mesong12b_drm.dtsi        | 21 ---------------------
 arch/arm/boot/dts/amlogic/mesonsm1_drm.dtsi         | 21 ---------------------
 arch/arm/boot/dts/amlogic/mesontl1_drm.dtsi         | 21 ---------------------
 arch/arm/boot/dts/amlogic/mesontm2_drm.dtsi         | 21 ---------------------
 .../dts/amlogic/sm1_s905d3_ac200_drm_buildroot.dts  |  2 +-
 .../dts/amlogic/sm1_s905x3_ac214_drm_buildroot.dts  |  2 +-
 .../amlogic/tm2_revb_t962e2_ab311_drm_buildroot.dts |  2 +-
 arch/arm/boot/dts/amlogic/tm2_t962e2_ab311_drm.dts  |  2 +-
 .../dts/amlogic/tm2_t962e2_ab311_drm_buildroot.dts  |  2 +-
 .../dts/amlogic/g12a_s905d2_u200_drm_buildroot.dts  |  2 +-
 .../dts/amlogic/g12a_s905x2_u212_drm_buildroot.dts  |  2 +-
 .../dts/amlogic/g12b_a311d_w400_drm_buildroot.dts   |  2 +-
 .../dts/amlogic/g12b_a311d_w400_drm_buildroot_a.dts |  2 +-
 arch/arm64/boot/dts/amlogic/mesong12a_drm.dtsi      | 21 ---------------------
 arch/arm64/boot/dts/amlogic/mesong12b_drm.dtsi      | 21 ---------------------
 arch/arm64/boot/dts/amlogic/mesonsm1_drm.dtsi       | 21 ---------------------
 arch/arm64/boot/dts/amlogic/mesontl1_drm.dtsi       | 21 ---------------------
 arch/arm64/boot/dts/amlogic/mesontm2_drm.dtsi       | 21 ---------------------
 .../dts/amlogic/sm1_s905d3_ac200_drm_buildroot.dts  |  2 +-
 .../dts/amlogic/sm1_s905x3_ac214_drm_buildroot.dts  |  2 +-
 .../arm64/boot/dts/amlogic/tm2_t962e2_ab311_drm.dts |  2 +-
 .../dts/amlogic/tm2_t962e2_ab311_drm_buildroot.dts  |  2 +-
 28 files changed, 18 insertions(+), 228 deletions(-)

diff --git a/arch/arm/boot/dts/amlogic/g12a_s905d2_u200_drm_buildroot.dts b/arch/arm/boot/dts/amlogic/g12a_s905d2_u200_drm_buildroot.dts
index 337aff3..2647933 100644
--- a/arch/arm/boot/dts/amlogic/g12a_s905d2_u200_drm_buildroot.dts
+++ b/arch/arm/boot/dts/amlogic/g12a_s905d2_u200_drm_buildroot.dts
@@ -821,7 +821,7 @@
 };
 
 &drm_amcvbsout {
-	status = "okay";
+	status = "disabled";
 };
 
 &drm_lcd {
diff --git a/arch/arm/boot/dts/amlogic/g12a_s905x2_u212_drm_buildroot.dts b/arch/arm/boot/dts/amlogic/g12a_s905x2_u212_drm_buildroot.dts
index 06cc532..11e6507 100644
--- a/arch/arm/boot/dts/amlogic/g12a_s905x2_u212_drm_buildroot.dts
+++ b/arch/arm/boot/dts/amlogic/g12a_s905x2_u212_drm_buildroot.dts
@@ -1451,7 +1451,7 @@
 };
 
 &drm_amcvbsout {
-	status = "okay";
+	status = "disabled";
 };
 
 &drm_lcd {
diff --git a/arch/arm/boot/dts/amlogic/g12b_a311d_w400_drm_buildroot.dts b/arch/arm/boot/dts/amlogic/g12b_a311d_w400_drm_buildroot.dts
index 9abe7ff..eac4059 100644
--- a/arch/arm/boot/dts/amlogic/g12b_a311d_w400_drm_buildroot.dts
+++ b/arch/arm/boot/dts/amlogic/g12b_a311d_w400_drm_buildroot.dts
@@ -1525,7 +1525,7 @@
 };
 
 &drm_amcvbsout {
-	status = "okay";
+	status = "disabled";
 };
 
 &drm_lcd {
diff --git a/arch/arm/boot/dts/amlogic/g12b_a311d_w400_drm_buildroot_a.dts b/arch/arm/boot/dts/amlogic/g12b_a311d_w400_drm_buildroot_a.dts
index 5cb634d..5315bb1 100644
--- a/arch/arm/boot/dts/amlogic/g12b_a311d_w400_drm_buildroot_a.dts
+++ b/arch/arm/boot/dts/amlogic/g12b_a311d_w400_drm_buildroot_a.dts
@@ -1490,7 +1490,7 @@
 };
 
 &drm_amcvbsout {
-	status = "okay";
+	status = "disabled";
 };
 
 &drm_lcd {
diff --git a/arch/arm/boot/dts/amlogic/g12b_revb_a311d_w400_drm_buildroot.dts b/arch/arm/boot/dts/amlogic/g12b_revb_a311d_w400_drm_buildroot.dts
index e4a18da..ffab95b 100644
--- a/arch/arm/boot/dts/amlogic/g12b_revb_a311d_w400_drm_buildroot.dts
+++ b/arch/arm/boot/dts/amlogic/g12b_revb_a311d_w400_drm_buildroot.dts
@@ -1463,7 +1463,7 @@
 };
 
 &drm_amcvbsout {
-	status = "okay";
+	status = "disabled";
 };
 
 &drm_lcd {
diff --git a/arch/arm/boot/dts/amlogic/mesong12a_drm.dtsi b/arch/arm/boot/dts/amlogic/mesong12a_drm.dtsi
index 99436ac..314e1ba 100644
--- a/arch/arm/boot/dts/amlogic/mesong12a_drm.dtsi
+++ b/arch/arm/boot/dts/amlogic/mesong12a_drm.dtsi
@@ -17,27 +17,6 @@
 #include <dt-bindings/display/meson-drm-ids.h>
 
 / {
-	venc-cvbs {
-		status = "okay";
-		compatible = "amlogic, meson-g12a-cvbs";
-
-		ports {
-			#address-cells = <1>;
-			#size-cells = <0>;
-
-			enc_cvbs_in: port@0 {
-				 #address-cells = <1>;
-				 #size-cells = <0>;
-				 reg = <0>;
-
-				 //venc_cvbs_in_vpu: endpoint@0 {
-				//	 reg = <0>;
-				//	 remote-endpoint = <&vpu_out_venc_cvbs>;
-				//};
-			};
-		};
-	};
-
 	drm_amhdmitx: drm-amhdmitx {
 		status = "disabled";
 		hdcp = "disabled";
diff --git a/arch/arm/boot/dts/amlogic/mesong12b_drm.dtsi b/arch/arm/boot/dts/amlogic/mesong12b_drm.dtsi
index 0e19ac1..40283bd 100644
--- a/arch/arm/boot/dts/amlogic/mesong12b_drm.dtsi
+++ b/arch/arm/boot/dts/amlogic/mesong12b_drm.dtsi
@@ -17,27 +17,6 @@
 #include <dt-bindings/display/meson-drm-ids.h>
 
 / {
-	venc-cvbs {
-		status = "okay";
-		compatible = "amlogic, meson-g12b-cvbs";
-
-		ports {
-			#address-cells = <1>;
-			#size-cells = <0>;
-
-			enc_cvbs_in: port@0 {
-				 #address-cells = <1>;
-				 #size-cells = <0>;
-				 reg = <0>;
-
-				 //venc_cvbs_in_vpu: endpoint@0 {
-				//	 reg = <0>;
-				//	 remote-endpoint = <&vpu_out_venc_cvbs>;
-				//};
-			};
-		};
-	};
-
 	drm_amhdmitx: drm-amhdmitx {
 		status = "disabled";
 		hdcp = "disabled";
diff --git a/arch/arm/boot/dts/amlogic/mesonsm1_drm.dtsi b/arch/arm/boot/dts/amlogic/mesonsm1_drm.dtsi
index 9c19b88..af43826 100644
--- a/arch/arm/boot/dts/amlogic/mesonsm1_drm.dtsi
+++ b/arch/arm/boot/dts/amlogic/mesonsm1_drm.dtsi
@@ -17,27 +17,6 @@
 #include <dt-bindings/display/meson-drm-ids.h>
 
 / {
-	venc-cvbs {
-		status = "okay";
-		compatible = "amlogic, meson-sm1-cvbs";
-
-		ports {
-			#address-cells = <1>;
-			#size-cells = <0>;
-
-			enc_cvbs_in: port@0 {
-				 #address-cells = <1>;
-				 #size-cells = <0>;
-				 reg = <0>;
-
-				 //venc_cvbs_in_vpu: endpoint@0 {
-				//	 reg = <0>;
-				//	 remote-endpoint = <&vpu_out_venc_cvbs>;
-				//};
-			};
-		};
-	};
-
 	drm_amhdmitx: drm-amhdmitx {
 		status = "disabled";
 		hdcp = "disabled";
diff --git a/arch/arm/boot/dts/amlogic/mesontl1_drm.dtsi b/arch/arm/boot/dts/amlogic/mesontl1_drm.dtsi
index 0974b21..4179beb 100644
--- a/arch/arm/boot/dts/amlogic/mesontl1_drm.dtsi
+++ b/arch/arm/boot/dts/amlogic/mesontl1_drm.dtsi
@@ -17,27 +17,6 @@
 #include <dt-bindings/display/meson-drm-ids.h>
 
 / {
-	venc-cvbs {
-		status = "okay";
-		compatible = "amlogic, meson-tl1-cvbs";
-
-		ports {
-			#address-cells = <1>;
-			#size-cells = <0>;
-
-			enc_cvbs_in: port@0 {
-				 #address-cells = <1>;
-				 #size-cells = <0>;
-				 reg = <0>;
-
-				 //venc_cvbs_in_vpu: endpoint@0 {
-				//	 reg = <0>;
-				//	 remote-endpoint = <&vpu_out_venc_cvbs>;
-				//};
-			};
-		};
-	};
-
 	drm_amhdmitx: drm-amhdmitx {
 		status = "disabled";
 		hdcp = "disabled";
diff --git a/arch/arm/boot/dts/amlogic/mesontm2_drm.dtsi b/arch/arm/boot/dts/amlogic/mesontm2_drm.dtsi
index d8337e2..f49a71c 100644
--- a/arch/arm/boot/dts/amlogic/mesontm2_drm.dtsi
+++ b/arch/arm/boot/dts/amlogic/mesontm2_drm.dtsi
@@ -17,27 +17,6 @@
 #include <dt-bindings/display/meson-drm-ids.h>
 
 / {
-	venc-cvbs {
-		status = "okay";
-		compatible = "amlogic, meson-tm2-cvbs";
-
-		ports {
-			#address-cells = <1>;
-			#size-cells = <0>;
-
-			enc_cvbs_in: port@0 {
-				 #address-cells = <1>;
-				 #size-cells = <0>;
-				 reg = <0>;
-
-				 //venc_cvbs_in_vpu: endpoint@0 {
-				//	 reg = <0>;
-				//	 remote-endpoint = <&vpu_out_venc_cvbs>;
-				//};
-			};
-		};
-	};
-
 	drm_amhdmitx: drm-amhdmitx {
 		status = "disabled";
 		hdcp = "disabled";
diff --git a/arch/arm/boot/dts/amlogic/sm1_s905d3_ac200_drm_buildroot.dts b/arch/arm/boot/dts/amlogic/sm1_s905d3_ac200_drm_buildroot.dts
index e466140..5e5ed4b 100644
--- a/arch/arm/boot/dts/amlogic/sm1_s905d3_ac200_drm_buildroot.dts
+++ b/arch/arm/boot/dts/amlogic/sm1_s905d3_ac200_drm_buildroot.dts
@@ -1162,7 +1162,7 @@
 };
 
 &drm_amcvbsout {
-	status = "okay";
+	status = "disabled";
 };
 
 &drm_lcd {
diff --git a/arch/arm/boot/dts/amlogic/sm1_s905x3_ac214_drm_buildroot.dts b/arch/arm/boot/dts/amlogic/sm1_s905x3_ac214_drm_buildroot.dts
index 44e280d..57d29f7 100644
--- a/arch/arm/boot/dts/amlogic/sm1_s905x3_ac214_drm_buildroot.dts
+++ b/arch/arm/boot/dts/amlogic/sm1_s905x3_ac214_drm_buildroot.dts
@@ -948,7 +948,7 @@
 };
 
 &drm_amcvbsout {
-	status = "okay";
+	status = "disabled";
 };
 
 &drm_lcd {
diff --git a/arch/arm/boot/dts/amlogic/tm2_revb_t962e2_ab311_drm_buildroot.dts b/arch/arm/boot/dts/amlogic/tm2_revb_t962e2_ab311_drm_buildroot.dts
index d16a440..67e22c9 100644
--- a/arch/arm/boot/dts/amlogic/tm2_revb_t962e2_ab311_drm_buildroot.dts
+++ b/arch/arm/boot/dts/amlogic/tm2_revb_t962e2_ab311_drm_buildroot.dts
@@ -1894,7 +1894,7 @@
 };
 
 &drm_amcvbsout {
-	status = "okay";
+	status = "disabled";
 };
 
 &drm_lcd {
diff --git a/arch/arm/boot/dts/amlogic/tm2_t962e2_ab311_drm.dts b/arch/arm/boot/dts/amlogic/tm2_t962e2_ab311_drm.dts
index 9dee1c1..4a3a4cf 100644
--- a/arch/arm/boot/dts/amlogic/tm2_t962e2_ab311_drm.dts
+++ b/arch/arm/boot/dts/amlogic/tm2_t962e2_ab311_drm.dts
@@ -1870,7 +1870,7 @@
 };
 
 &drm_amcvbsout {
-	status = "okay";
+	status = "disabled";
 };
 
 &drm_lcd {
diff --git a/arch/arm/boot/dts/amlogic/tm2_t962e2_ab311_drm_buildroot.dts b/arch/arm/boot/dts/amlogic/tm2_t962e2_ab311_drm_buildroot.dts
index 7857bf6..606584c 100644
--- a/arch/arm/boot/dts/amlogic/tm2_t962e2_ab311_drm_buildroot.dts
+++ b/arch/arm/boot/dts/amlogic/tm2_t962e2_ab311_drm_buildroot.dts
@@ -1871,7 +1871,7 @@
 };
 
 &drm_amcvbsout {
-	status = "okay";
+	status = "disabled";
 };
 
 &drm_lcd {
diff --git a/arch/arm64/boot/dts/amlogic/g12a_s905d2_u200_drm_buildroot.dts b/arch/arm64/boot/dts/amlogic/g12a_s905d2_u200_drm_buildroot.dts
index 17087c0..8a69163 100644
--- a/arch/arm64/boot/dts/amlogic/g12a_s905d2_u200_drm_buildroot.dts
+++ b/arch/arm64/boot/dts/amlogic/g12a_s905d2_u200_drm_buildroot.dts
@@ -1477,7 +1477,7 @@
 };
 
 &drm_amcvbsout {
-	status = "okay";
+	status = "disabled";
 };
 
 &drm_lcd {
diff --git a/arch/arm64/boot/dts/amlogic/g12a_s905x2_u212_drm_buildroot.dts b/arch/arm64/boot/dts/amlogic/g12a_s905x2_u212_drm_buildroot.dts
index d1bd45c..804cd24 100644
--- a/arch/arm64/boot/dts/amlogic/g12a_s905x2_u212_drm_buildroot.dts
+++ b/arch/arm64/boot/dts/amlogic/g12a_s905x2_u212_drm_buildroot.dts
@@ -819,7 +819,7 @@
 };
 
 &drm_amcvbsout {
-	status = "okay";
+	status = "disabled";
 };
 
 &drm_lcd {
diff --git a/arch/arm64/boot/dts/amlogic/g12b_a311d_w400_drm_buildroot.dts b/arch/arm64/boot/dts/amlogic/g12b_a311d_w400_drm_buildroot.dts
index 966d0c5..44a7808 100644
--- a/arch/arm64/boot/dts/amlogic/g12b_a311d_w400_drm_buildroot.dts
+++ b/arch/arm64/boot/dts/amlogic/g12b_a311d_w400_drm_buildroot.dts
@@ -908,7 +908,7 @@
 };
 
 &drm_amcvbsout {
-	status = "okay";
+	status = "disabled";
 };
 
 &drm_lcd {
diff --git a/arch/arm64/boot/dts/amlogic/g12b_a311d_w400_drm_buildroot_a.dts b/arch/arm64/boot/dts/amlogic/g12b_a311d_w400_drm_buildroot_a.dts
index f92ab2b..05486cc 100644
--- a/arch/arm64/boot/dts/amlogic/g12b_a311d_w400_drm_buildroot_a.dts
+++ b/arch/arm64/boot/dts/amlogic/g12b_a311d_w400_drm_buildroot_a.dts
@@ -875,7 +875,7 @@
 };
 
 &drm_amcvbsout {
-	status = "okay";
+	status = "disabled";
 };
 
 &drm_lcd {
diff --git a/arch/arm64/boot/dts/amlogic/mesong12a_drm.dtsi b/arch/arm64/boot/dts/amlogic/mesong12a_drm.dtsi
index 37e04fa..8d97c3a 100644
--- a/arch/arm64/boot/dts/amlogic/mesong12a_drm.dtsi
+++ b/arch/arm64/boot/dts/amlogic/mesong12a_drm.dtsi
@@ -17,27 +17,6 @@
 #include <dt-bindings/display/meson-drm-ids.h>
 
 / {
-	venc-cvbs {
-		status = "okay";
-		compatible = "amlogic, meson-g12a-cvbs";
-
-		ports {
-			#address-cells = <1>;
-			#size-cells = <0>;
-
-			enc_cvbs_in: port@0 {
-				 #address-cells = <1>;
-				 #size-cells = <0>;
-				 reg = <0>;
-
-				 //venc_cvbs_in_vpu: endpoint@0 {
-				//	 reg = <0>;
-				//	 remote-endpoint = <&vpu_out_venc_cvbs>;
-				//};
-			};
-		};
-	};
-
 	drm_amhdmitx: drm-amhdmitx {
 		status = "disabled";
 		hdcp = "disabled";
diff --git a/arch/arm64/boot/dts/amlogic/mesong12b_drm.dtsi b/arch/arm64/boot/dts/amlogic/mesong12b_drm.dtsi
index 41ce4a7..2a8f5b6 100644
--- a/arch/arm64/boot/dts/amlogic/mesong12b_drm.dtsi
+++ b/arch/arm64/boot/dts/amlogic/mesong12b_drm.dtsi
@@ -17,27 +17,6 @@
 #include <dt-bindings/display/meson-drm-ids.h>
 
 / {
-	venc-cvbs {
-		status = "okay";
-		compatible = "amlogic, meson-g12b-cvbs";
-
-		ports {
-			#address-cells = <1>;
-			#size-cells = <0>;
-
-			enc_cvbs_in: port@0 {
-				 #address-cells = <1>;
-				 #size-cells = <0>;
-				 reg = <0>;
-
-				 //venc_cvbs_in_vpu: endpoint@0 {
-				//	 reg = <0>;
-				//	 remote-endpoint = <&vpu_out_venc_cvbs>;
-				//};
-			};
-		};
-	};
-
 	drm_amhdmitx: drm-amhdmitx {
 		status = "disabled";
 		hdcp = "disabled";
diff --git a/arch/arm64/boot/dts/amlogic/mesonsm1_drm.dtsi b/arch/arm64/boot/dts/amlogic/mesonsm1_drm.dtsi
index 43de163..79039d2 100644
--- a/arch/arm64/boot/dts/amlogic/mesonsm1_drm.dtsi
+++ b/arch/arm64/boot/dts/amlogic/mesonsm1_drm.dtsi
@@ -17,27 +17,6 @@
 #include <dt-bindings/display/meson-drm-ids.h>
 
 / {
-	venc-cvbs {
-		status = "okay";
-		compatible = "amlogic, meson-sm1-cvbs";
-
-		ports {
-			#address-cells = <1>;
-			#size-cells = <0>;
-
-			enc_cvbs_in: port@0 {
-				 #address-cells = <1>;
-				 #size-cells = <0>;
-				 reg = <0>;
-
-				 //venc_cvbs_in_vpu: endpoint@0 {
-				//	 reg = <0>;
-				//	 remote-endpoint = <&vpu_out_venc_cvbs>;
-				//};
-			};
-		};
-	};
-
 	drm_amhdmitx: drm-amhdmitx {
 		status = "disabled";
 		hdcp = "disabled";
diff --git a/arch/arm64/boot/dts/amlogic/mesontl1_drm.dtsi b/arch/arm64/boot/dts/amlogic/mesontl1_drm.dtsi
index 2ca57a4..1b6f03a 100644
--- a/arch/arm64/boot/dts/amlogic/mesontl1_drm.dtsi
+++ b/arch/arm64/boot/dts/amlogic/mesontl1_drm.dtsi
@@ -17,27 +17,6 @@
 #include <dt-bindings/display/meson-drm-ids.h>
 
 / {
-	venc-cvbs {
-		status = "okay";
-		compatible = "amlogic, meson-tl1-cvbs";
-
-		ports {
-			#address-cells = <1>;
-			#size-cells = <0>;
-
-			enc_cvbs_in: port@0 {
-				 #address-cells = <1>;
-				 #size-cells = <0>;
-				 reg = <0>;
-
-				 //venc_cvbs_in_vpu: endpoint@0 {
-				//	 reg = <0>;
-				//	 remote-endpoint = <&vpu_out_venc_cvbs>;
-				//};
-			};
-		};
-	};
-
 	drm_amhdmitx: drm-amhdmitx {
 		status = "disabled";
 		hdcp = "disabled";
diff --git a/arch/arm64/boot/dts/amlogic/mesontm2_drm.dtsi b/arch/arm64/boot/dts/amlogic/mesontm2_drm.dtsi
index 5e5a8de..303c953 100644
--- a/arch/arm64/boot/dts/amlogic/mesontm2_drm.dtsi
+++ b/arch/arm64/boot/dts/amlogic/mesontm2_drm.dtsi
@@ -17,27 +17,6 @@
 #include <dt-bindings/display/meson-drm-ids.h>
 
 / {
-	venc-cvbs {
-		status = "okay";
-		compatible = "amlogic, meson-tm2-cvbs";
-
-		ports {
-			#address-cells = <1>;
-			#size-cells = <0>;
-
-			enc_cvbs_in: port@0 {
-				 #address-cells = <1>;
-				 #size-cells = <0>;
-				 reg = <0>;
-
-				 //venc_cvbs_in_vpu: endpoint@0 {
-				//	 reg = <0>;
-				//	 remote-endpoint = <&vpu_out_venc_cvbs>;
-				//};
-			};
-		};
-	};
-
 	drm_amhdmitx: drm-amhdmitx {
 		status = "disabled";
 		hdcp = "disabled";
diff --git a/arch/arm64/boot/dts/amlogic/sm1_s905d3_ac200_drm_buildroot.dts b/arch/arm64/boot/dts/amlogic/sm1_s905d3_ac200_drm_buildroot.dts
index b7bccf3..0f7d48c 100644
--- a/arch/arm64/boot/dts/amlogic/sm1_s905d3_ac200_drm_buildroot.dts
+++ b/arch/arm64/boot/dts/amlogic/sm1_s905d3_ac200_drm_buildroot.dts
@@ -1160,7 +1160,7 @@
 };
 
 &drm_amcvbsout {
-	status = "okay";
+	status = "disabled";
 };
 
 &drm_lcd {
diff --git a/arch/arm64/boot/dts/amlogic/sm1_s905x3_ac214_drm_buildroot.dts b/arch/arm64/boot/dts/amlogic/sm1_s905x3_ac214_drm_buildroot.dts
index 18dd81c..1d7cef0 100644
--- a/arch/arm64/boot/dts/amlogic/sm1_s905x3_ac214_drm_buildroot.dts
+++ b/arch/arm64/boot/dts/amlogic/sm1_s905x3_ac214_drm_buildroot.dts
@@ -1026,7 +1026,7 @@
 };
 
 &drm_amcvbsout {
-	status = "okay";
+	status = "disabled";
 };
 
 &drm_lcd {
diff --git a/arch/arm64/boot/dts/amlogic/tm2_t962e2_ab311_drm.dts b/arch/arm64/boot/dts/amlogic/tm2_t962e2_ab311_drm.dts
index 384b76d..ae59421 100644
--- a/arch/arm64/boot/dts/amlogic/tm2_t962e2_ab311_drm.dts
+++ b/arch/arm64/boot/dts/amlogic/tm2_t962e2_ab311_drm.dts
@@ -1866,7 +1866,7 @@
 };
 
 &drm_amcvbsout {
-	status = "okay";
+	status = "disabled";
 };
 
 &drm_lcd {
diff --git a/arch/arm64/boot/dts/amlogic/tm2_t962e2_ab311_drm_buildroot.dts b/arch/arm64/boot/dts/amlogic/tm2_t962e2_ab311_drm_buildroot.dts
index 6a9e042..36d8407 100644
--- a/arch/arm64/boot/dts/amlogic/tm2_t962e2_ab311_drm_buildroot.dts
+++ b/arch/arm64/boot/dts/amlogic/tm2_t962e2_ab311_drm_buildroot.dts
@@ -1867,7 +1867,7 @@
 };
 
 &drm_amcvbsout {
-	status = "okay";
+	status = "disabled";
 };
 
 &drm_lcd {
-- 
2.7.4

